tests/testapps/AddressBookSample/AddressBook/ViewAllContactsViewController.m

```diff
@@ -41,10 +41,30 @@ - (void)viewDidLoad {
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
 
+#ifdef WINOBJC
```

Comment:
I will document this in the code as well, but it's because we added a new method (ABAddressBookCopyArrayOfAllUserAppPeople) to get modifiable/delete-able contacts due to the Windows permissions issue (but this method doesn't exist on the iOS side, so the ifdef is so the code can still work on iOS)

---
In reply to: [77093056](https://github.com/Microsoft/WinObjC/pull/935#discussion_r77093056) [](ancestors = 77093056)